src/main/java/ch/enterag/utils/mime/MimeTypes.java

```diff
@@ -1,788 +1,20 @@
 package ch.enterag.utils.mime;
 
-import java.util.*;
+import org.apache.tika.mime.MimeTypeException;
+
+import static org.apache.tika.mime.MimeTypes.getDefaultMimeTypes;
+
+public class MimeTypes {
+
+    public static String getExtension(String mimeType) throws MimeTypeException {
```

Comment:
The public API should not change. Instead of throwing a MimeTypeException, handle the exception in the method itself. The default exception should be returned, if an invalid mime type was provided. 

I've added an assertion to MimeTypesTest.